Спецификация для API

RO1T opened the pull request:
Добавлена спецификация open api

---
:mortar_board: [Спецификация для API](https://up.htmlacademy.ru/univer-nodejs-api/3/user/2220577/tasks/8)

RO1T committed:
5.8. Спецификация для API

Добавлена спецификация open api

polRk commented on specification/specification.yml:
```diff
@@ -0,0 +1,948 @@
+
+paths:
+  /offer/create:
+    post:
```
Такой ручки нет у фронтенда. Сделайте, пожалуйста, спецификацию так, чтобы она соответствовала, фронтенду

RO1T committed:
Спецификация, доработка

изменена спецификация, следуя апи фронта

https://github.com/htmlacademy-nodejs-api/frontend-six-cities/tree/main
